_includes/parse-server/third-party-auth.md

```diff
@@ -59,6 +59,18 @@ Note, most of them don't require a server configuration so you can use them dire
   }
 }
 ```
+
+The options passed to Parse Server:
+```js
+{
+  auth: {
+    facebook: {
+      appIds: "", // can be a string or an array of strings
```

Comment:
`appIds: ["id1", "id2"],`

Could you also describe what this parameter does? And whether it is optional or not (I think it is optional)

Reply:
done